CONTRIBUTORS.md

```diff
@@ -76,7 +76,5 @@ submit the change with your pull request.
 - Brenda Wallace / [br3nda](https://github.com/br3nda)
 - Jim Stallings / [jestallin](https://github.com/jestallin)
 - Alyssa Ransbury / [alran](https://github.com/alran)
-
-## Bots
-
+- Thomas Countz / [thomascountz](https://github.com/thomascountz)
```

Comment:
yeah, if you `git rebase` you'll get a git conflict which you can resolve and re-instate those lines.